src/configs/react.js

```diff
@@ -6,8 +6,9 @@ import reactHooks from 'eslint-plugin-react-hooks';
 import globals from 'globals';
 import tseslint from 'typescript-eslint';
 import { REACT_FILE_PATTERNS } from '../lib/patterns.js';
+import javascript from './javascript.js';
 
-export default tseslint.config({
+export default tseslint.config(...javascript, {
```

Comment:
❔ I wonder if there may be some JS rules that might yield false positives for React and therefore might need to be turned off. Did you check this?

Reply:
I'm not aware of any such rules. But I'll add them if they come up.